go-controller/pkg/config/config.go

```diff
@@ -2124,13 +2128,39 @@ func completeDefaultConfig(allSubnets *ConfigSubnets) error {
 		allSubnets.Append(ConfigSubnetCluster, subnet.CIDR)
 	}
 
+	Default.UDNAllowedDefaultServices, err = parseServicesNamespacedNames(Default.RawUDNAllowedDefaultServices)
```

Comment:
In a scenario of parse fails, `config.Default` will be `nil`.
Other component may read  `config`, right? 
Is so do you think we should change it to not override `Default` in case of an error?

Reply:
`Default.UDNAllowedDefaultServices` will be `nil` in case the parse fails but this is fine for a slice.
Additionally this is done at startup so a failure to parse `RawUDNAllowedDefaultServices` will cause ovn-k to exit early.